pages.pt_BR/common/duc.md

```diff
@@ -0,0 +1,28 @@
+# duc
+
+> Duc é uma coleção de ferramentas para indexar, inspecionar e visualizar uso do disco. O duc mantém uma base de dados dos tamanhos acumlados dos diretórios do sistema de arquivos, permitindo buscas nessa base, ou a criação de gráficos elegantes.
+> Mais informações: <https://duc.zevv.nl/>.
+
+- Indexa o diretório /usr, escrevendo a base de dados para o local default em ~/.duc.db:
+
+`duc index {{/usr}}`
+
+- Lista todos os arqivos e diretórios dentro do /usr/local, mostrando tamanho relativo dos arquivos em um [g]ráfico:
```

Comment:
```suggestion
- Lista todos os arqivos e diretórios dentro do /usr/local, mostrando tamanho relativo dos arquivos em um [g]raph (gráfico):
```

We use mnemonics interchangeably to highlight long/short options. We can use the English word here and wrap the translation inside brackets.